From 8d6d533394b993bf98440651a4ab90f687e84197 Mon Sep 17 00:00:00 2001
From: Pankaj Bharadiya <pankaj.laxminarayan.bharadiya@intel.com>
Date: Tue, 26 Feb 2019 13:30:02 +0530
Subject: [PATCH 1036/1214] ASoC: Intel: Skylake: Virt: Fix build error
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Remove unwanted extra argument to fix build error.

sound/soc/intel/skylake/virtio/skl-virt-audio.c: In function ‘vskl_vbs_handle_kick’:
sound/soc/intel/skylake/virtio/skl-virt-audio.c:101:3: error: too many arguments to function ‘acrn_ioreq_complete_request’
   acrn_ioreq_complete_request(client->vhm_client_id, vcpu, req);
   ^

Change-Id: Ifed5194d71cf96d83d194ea6e619e302fb95c1a7
Tracked-On: OAM-76301
Signed-off-by: Pankaj Bharadiya <pankaj.laxminarayan.bharadiya@intel.com>
---
 sound/soc/intel/skylake/virtio/skl-virt-audio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/intel/skylake/virtio/skl-virt-audio.c b/sound/soc/intel/skylake/virtio/skl-virt-audio.c
index 0b712db..2aec728 100644
--- a/sound/soc/intel/skylake/virtio/skl-virt-audio.c
+++ b/sound/soc/intel/skylake/virtio/skl-virt-audio.c
@@ -98,7 +98,7 @@ static int vskl_vbs_handle_kick(int client_id, unsigned long *ioreqs_map)
 				(handle = 1) : (handle = 0);
 		}
 
-		acrn_ioreq_complete_request(client->vhm_client_id, vcpu, req);
+		acrn_ioreq_complete_request(client->vhm_client_id, vcpu);
 
 		/* handle VQ kick if needed */
 		if (handle)
-- 
2.7.4

